src/engraving/rendering/dev/slurtielayout.cpp

```diff
@@ -531,6 +552,18 @@ void SlurTieLayout::slurPos(Slur* item, SlurPos* sp, LayoutContext& ctx)
     sp->p1 = scr->pos() + scr->segment()->pos() + scr->measure()->pos();
     sp->p2 = ecr->pos() + ecr->segment()->pos() + ecr->measure()->pos();
 
+    if (item->staffType() && item->staffType()->isJianpu()) {
+        double symWidth = item->symWidth(SymId::keysigJianpuDo);
+        double symHeight = item->symHeight(SymId::keysigJianpuDo);
```

Comment:
In situations like this, it's more efficient to call `symBbox` once, and then get the width and height from that rectangle. That's more efficient because `symWidth` and `symHeight` both call `symBbox` internally, so if you call both of them separately, the bbox calculation has to be done twice, which is a bit expensive.